src/IntelliTect.Coalesce/TypeDefinition/MethodViewModel.cs

```diff
@@ -49,9 +50,21 @@ internal MethodViewModel(ClassViewModel parent)
         public string JsVariableModal => JsVariable + "Modal";
         public string JsVariableArgs => JsVariable + "Args";
         public string JsVariableWithArgs => JsVariable + "WithArgs";
+        public string HttpMethodName => HttpMethodType.ToString().ToUpper();
+        public string ApiControllerAnnotation => $"Http{HttpMethodType.ToString()}";
```

Comment:
Recommend renaming `ApiControllerAnnotation` to something like `ApiActionHttpMethodAnnotation` so its purpose is clear.

Reply:
Done